catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -181,6 +192,29 @@ public Metacard getMetacard(String id)
             }
             resultTypes.put(metacardType.getName(), attributes);
         }
+        for (InjectableAttribute attribute : injectableAttributes) {
```

Comment:
It seems like this block and the one above are very similar.  Can we move this to a common method?